app/index.html

```diff
@@ -469,11 +469,11 @@ <h2 class="title-margin-bottom">Media</h2>
        <div class="row">
            <div class="twelve columns">
                <div class="contact-icon">
-                   <a href="mailto:contact.cornellfencingassociation@outlook.com" target="_blank"><i class="fa fa-envelope-o"></i></a>
+                   <a href="mailto:contact@cornellfencingassociation.com" target="_blank"><i class="fa fa-envelope-o"></i></a>

```

Comment:
We actually don't want `target="_blank"` on `mailto` links, because it opens in a new tab with the `mailto` url, and also opens in your mail client. If we leave off the `target="_blank"`, it'll open in the mail client, but won't take the user away from the page, which is what we want.